templates/zerver/help/scim.md

```diff
@@ -1,62 +1,88 @@
 
-1. Continue to `Sign-On Options`. Leave the `SAML` options, as this type of Okta application
-   doesn't actually support `SAML` authentication, and you'll need to set up a separate Okta app
-   to activate `SAML` for your Zulip organization.
+1. Click **Add** and choose your **Application label**. For example, you can
+   name it "Zulip SCIM".
 
-1. In `Credentials Details`, set `Application username format` to `Email` and
-    `Update application username on` to `Create and update`.
+1. Continue to **Sign-On Options**. Leave the **SAML** options as they are.
+   This type of Okta application doesn't actually support SAML authentication,
+   and you'll need to set up a separate Okta app to activate SAML for your Zulip
+   organization.
 
-1. The Okta app has been added. Navigate to the `Provisioning` tab.
+1. In **Credentials Details**, specify the following fields:
+     * **Application username format**: `Email`
+     * **Update application username on**: `Create and update`
 
-1. Click `Configure API Integration` and check the `Enable API integration` box.
-   Okta will ask you for the `Base URL` and `API token`. The `Base URL` should be
-   `yourorganization.zulipchat.com/scim/v2` and for `API token` you'll set the value
-   given to you by support. When you proceed to the next step, Okta will verify that
-   these details are correct by making a SCIM request to the Zulip server.
+1. In the **Provisioning** tab, click **Configure API Integration**, check the
+   **Enable API integration** checkbox, and specify the following fields:
+     * **Base URL**: `yourorganization.zulipchat.com/scim/v2`
+     * **API token**: `Bearer token` (given to you by Zulip support)
 
-1. In the `To App` section of the `Provisioning` tab (which should be opened by default
-   when you continue from the previous step), edit the `Provisioning to App` settings
-   to enable `Create Users`, `Update User Attributes` and `Deactivate Users`.
+    When you proceed to the next step, Okta will verify that these details are
+    correct by making a SCIM request to the Zulip server.
 
-1. In `Attribute Mappings`, remove all attributes except `userName`, `givenName`
-   and `familyName`.
+1. Enable the following **Provisioning to App** settings:
+     * **Create Users**
+     * **Update User Attributes**
+     * **Deactivate Users**
 
-1. Now the integration should be ready and you can `Assign` users to
-   the app to configure their Zulip accounts to be managed by
-   SCIM. When you assign a user, Okta will check if the account exists
-   in your Zulip organization and if it doesn't, the account will be
-   created. Changes to the user's email or name in Okta will
+1. Remove all attributes in **Attribute Mappings**, _except_ for the following:
+     * **userName**
+     * **givenName**
+     * **familyName**
+
+1. **Assign** users to the app now that the integration is ready to configure
+   their Zulip accounts to be managed by SCIM. When you assign a user, Okta will
```

Comment:
```
**Assign** users to the app now that the integration is ready to manage
   their Zulip accounts by SCIM.
```

to simplify the sentence a bit? 

Looks great aside of this one small tweak!

Reply:
Now it sounds a little ambiguous to me. I would add "Zulip" and "SCIM" to the phrase. What do you think?

> Now that the integration is ready to manage Zulip user accounts, **assign** users to the SCIM app.

I am assuming **assign** is a label in the Okta UI.

Reply:
Works for me. @mateuszmandera could you double-check this one?